hack/verify-codegen.sh

```diff
@@ -52,9 +52,8 @@ diff -Naupr --no-dereference \
 diff -Naupr --no-dereference \
   "${REPO_ROOT_DIR}/pkg" "${TMP_DIFFROOT}/pkg" || ret=1
 
-# TODO uncomment this once we fix https://github.com/google/knative-gcp/issues/702
-# diff -Naupr --no-dereference \
-#  "${REPO_ROOT_DIR}/third_party" "${TMP_DIFFROOT}/third_party" || ret=1
+diff -Naupr --no-dereference \
```

Comment:
I see #702 is still open. Is it fixed?

Reply:
Sorry I didn't notice this PR actually fixes it

Reply:
Yeah it was fixed a while ago.